text/src/autogluon/text/text_prediction/text_presets.py

```diff
@@ -0,0 +1,54 @@
+from ..automm.presets import get_preset
+
+
+def list_presets():
+    """
+    Returns
+    -------
+    A list of naive presets available in AutoGluon-Text.
+    """
+    simple_presets = [
+        "default", "lower_quality_fast_train",
+        "medium_quality_faster_train", "best_quality",
+    ]
+    return simple_presets
```

Comment:
Consider to use the registry design later.https://github.com/awslabs/autogluon/blob/master/text/src/autogluon/text/text_prediction/presets.py You may directly copy `registry.py` in https://github.com/sxjscience/autogluon-contrib-nlp/blob/master/src/autogluon_contrib_nlp/utils/registry.py to AutoGluon. The corresponding test-case is in https://github.com/dmlc/gluon-nlp/blob/master/tests/test_utils_registry.py .